src/lib/libdylink.js

```diff
@@ -48,6 +48,62 @@ var LibraryDylink = {
     registerWasmPlugin();
     `,
   $preloadedWasm: {},
+
+  $locateLibraryFromFS__deps: ['$FS', '$PATH'],
+  $locateLibraryFromFS: (filename, searchDirs) => {
+    // Find the library in the filesystem
+    // returns [true, abspath] if found, [false, null] otherwise.
+    if (!(typeof FS.findObject === 'function')) {
```

Comment:
I think we can use `#ifdef` to stub out this function, but I'm not sure what the right test is to check if FS is included. Probably this check can replace the FS dependency.

Reply:
> I also removed the FS and PATH dependency as I think they are implied in the ifdef FILESYSTEM.

Never mind. It doesn't seem to work in some cases.